emission/exportdata/export_data.py

```diff
@@ -33,6 +33,6 @@ def last_trip_done(self):
     def run_export_data_pipeline(self, user_id):
         ts = esta.TimeSeries.get_time_series(user_id)
         time_query = espq.get_time_range_for_export_data(user_id)
-        file_name = "archive_%s_%s_%s" % (user_id, time_query.startTs, time_query.endTs)
+        file_name = "emission/archived/archive_%s_%s_%s" % (user_id, time_query.startTs, time_query.endTs)
```

Comment:
I would not put the output files into the source directory. It is very tempting to do so, but it then generates a bunch of files which are not under source control in the git repository, which is not a great idea. You have a config for the export (e.g.  `conf/log/export.conf.sample`); why not put a default path in there and allow the user to override it?

Reply:
CONFIG_DATA_DIR + "" ? 
Talked with Kyle briefly about this but still unsure of where this would fall into the conf/log/export.conf.sample file. How would we put in a default path, and what would allow the user to override it?

Reply:
My ideas for this: 
If we declare data_dir = emission/archived we can then just write as the file_name: data_dir + "archive...". Would this work? 
Additionally, this data_dir would need to be declared within the conf/log/export.conf.sample file. Where does this get put and what will it look like? I have seen multiple examples on various open source projects but am unsure what is going to work here.

Reply:
seems like you can just have

```
{
    "export_output_dir": "<default path>",
   "handlers": {....}
}
```

is there a reason why that doesn't work?

Or you can assume an environment variable so it is easier to use with docker-compose.

Reply:
> Where does this get put and what will it look like? I have seen multiple examples on various open source projects but am unsure what is going to work here.

Maybe you can try out some of those approaches and let us know why they don't work.

Reply:
following up on the "export_output_dir" option, how is this used? 

file_name = export_output_dir + "archive_%s_%s_%s" % (self.testUUID, time_query.startTs, time_query.endTs)
NameError: name 'export_output_dir' is not defined

Above is the attempted code I tried to run. Do we need to import the conf/logs/ file? or is this not what you intended?

Reply:
> Do we need to import the conf/logs/ file?

Please look at other examples of config files in the e-mission server code. You need to read the file in (which you already do in the intake stage). There's no magic to define new variables from a config file.

Also, I thought you were going to use environment variables since it is easier to configure with `docker-compose.yml`

Reply:
Sorry I had mentioned both yesterday, but said to keep your convention. @jruzekowicz in this case, you can look at my [ei-sync-server](https://github.com/hexmap/ei-sync-server/) config file to see how that can work with Docker.

Reply:
Sounds good! @kafitz I see that you have a DATA_DIR environment variable within your docker compose file. 
I am thinking of putting in something such as: os.environ.get('DATA_DIR', 'emission/archived') that way it defaults to the emission/archived folder on my side of things, and would use the environment variable you have already set. 

Is this what we were thinking? This still seems to hard code things in. I am having a little confusion on where I would set an environment variable on my end, or if it is needed if I can just have the default fallback path. Is this the right direction? My understanding is that unless the DATA_DIR environment variable has been set, it will fall back on the path I directed. This gives users the ability to change the storage location. 

I was looking through you docker-compose.yml file and this appeared to be consistent with the style and application used. Am I missing anything here?

Reply:
This is the right direction. Bring your questions to the 1:1.

Reply:
That seems good, I think it's fine to hardcode the default path here for now. Sometimes you may want to set it to None and check that a value exists when the script starts, especially something sensitive like a password salt.

For running Python scripts normally, you can set an env value on-the-fly like `DEBUG=true python myscript.py`. For docker, we'll configure all through the docker-compose.yml like you see.

Reply:
Okay, I will push another commit so we have code to look at. @kafitz I will add you to the meeting if you have time/want to join.